src/features/Send/useCases/ListAmountsToSend/AddToken/SelectTokenFromListScreen.tsx

```diff
@@ -130,4 +130,59 @@ const messages = defineMessages({
     id: 'components.send.assetselectorscreen.unknownAsset',
     defaultMessage: '!!!Unknown asset',
   },
+  counter1: {
+    id: 'components.send.assetselectorscreen.counter1',
```

Comment:
`global.txLabels.assets` ?

Reply:
This is a temporary solution until the counter is fully implemented.

Reply:
removed in this pr https://github.com/Emurgo/yoroi-mobile/pull/2426

it will be fully implemented in other PR related with its tickets